vizro-ai/mkdocs.yml

```diff
@@ -13,13 +13,15 @@ nav:
           - Advanced charts: pages/user-guides/create-advanced-charts.md
           - Different languages: pages/user-guides/use-different-languages.md
           - Vizro-AI charts in dashboards: pages/user-guides/add-generated-chart-usecase.md
+  - API Reference:
+      - VizroAI: pages/API-reference/vizro-ai.md
   - Explanation:
       - FAQs: pages/explanation/faq.md
       - Disclaimer: pages/explanation/disclaimer.md
       - Safeguard code execution: pages/explanation/safeguard.md
       - Safety in Vizro-AI: pages/explanation/safety-in-vizro-ai.md
-  - API Reference:
-      - VizroAI: pages/API-reference/vizro-ai.md
+  - Examples:
```

Comment:
```suggestion
  - Gallery:
```

Reply:
I think we should keep this as examples to align with the Vizro docs, and because it is an example that people can start from, and not just pictures they can look at.